project-reports/2023/2023-Q4-Hyperledger-Indy.md

```diff
@@ -0,0 +1,173 @@
+- indy-node-container - v1.2.5
+- indy-shared-rs - v0.3.3
+
+## Overall Activity in the Past Quarter
+
+GitHub activity was focused on the so-called Shared Components ([indy-vdr],
+[indy-shared-rs] and [aries-askar]) and the transition away from the Indy SDK. The
+maintainer community has discussed the formal deprecation of the Indy SDK and
+the Aries maintainers have been busy making the Shared Components the default in
+Aries, in preparation for the deprecation announcement.  
+
+At the Second Indy Ecosystem Summit, two discussions focused on paths forward.
+On the technical side, [DSR Corporation](https://en.dsr-corporation.com/)
+presented several ideas for evolving the Indy Node ledger component, including
+evolving the current implementation, or implementing Indy transactions on
+another network, such as [Hyperledger Besu](https://www.hyperledger.org/projects/besu), or
+[Cosmos](https://cosmos.network/). While there is no pressing feature set needed in the
+foreseeable future, the lack of maintainers on the existing code base is concerning. Those
+with an opinion thought that Besu was the most interesting alternative and work has
+started on defining what that might mean. This breaks into a couple of questions:
+
+* How to write Indy (or more accurately, AnonCreds) objects on a Besu network
+  (schema, credential definitions, and revocation registries)?
+* Can and should the existing Indy roles be used when running on a Besu network?
+* Can and should the existing Indy authorization rules be used when running on a Besu network?
+
+The second topic raised was if and how the Indy community, as represented at the
+Ecosystems Summits, can collaborate on moving the Indy technology, marketing and
+regulatory environment forward?  For example, is there a way for the Hyperledger
+Foundation to enable a group to collect funds for an open source objective, and
+the use those funds to enable a code contribution?
+
+## Current Plans
+
+Continuing the move of the Aries Frameworks to [indy-vdr] and [aries-askar] and away
+from [indy-sdk], which will enable its deprecation. The high priority work on
+[Hyperledger AnonCreds] implementation will also drive that effort.
+
+Investigating different paths forward for Hyperledger Indy ledger components, including
+a specific look at the use of Hyperledger Besu for that purpose. What would that look
+like as a technical component?
+
+A third Hyperledger Indy Ecosystem Summit is planned, but not yet scheduled, likely
+with a more focused group -- those that are operating Indy networks.
+
+## Maintainer Diversity
+
+There are 35 individuals on 22 Indy GitHub Teams representing at least 16 organizations.
+
+Still needed -- a cleanup of the teams are needed as some of the individuals are
+no longer active in the community. No progress was made on that cleanup this
+quarter. We have the team lists, but have not queried those on the list to see
+if they are still interested in being maintainers.
```

Comment:
Is there a Indy-specific maintainer policy, or could we use the [Maintainer Inactivity Policy](https://toc.hyperledger.org/governing-documents/inactivity.html#maintainer-inactivity) that the TOC previously approved for determining when the last action was taken and removing anyone that has not had activity in the past 6 months?

Reply:
Yes.  It’s just been time and priorities that has prevented this from being done...